src/ansys/motorcad/core/geometry.py

```diff
@@ -464,8 +483,113 @@ def find_entity_from_coordinates(self, coordinate_1, coordinate_2):
         return None
 
 
+class RegionMagnet(Region):
+    """Provides the Python representation of a Motor-CAD magnet geometry region."""
+
+    def __init__(self, motorcad_instance=None):
+        """Initialise a ``RegionMagnet`` instance."""
+        super().__init__(motorcad_instance)
+        self._magnet_angle = 0.0
+        self._br_multiplier = 0.0
+        self._br_magnet = 0.0
+        self._magnet_polarity = ""
+
+    def _to_json(self):
+        """Convert from a Python class to a JSON object.
+
+        Returns
+        -------
+        dict
+            Dictionary of the geometry region represented as JSON.
+        """
+        region_dict = super()._to_json()
+
+        region_dict["magnet_magfactor"] = self._br_multiplier
+        region_dict["magnet_angle"] = self._magnet_angle
+
+        return region_dict
+
+    @property
+    def br_multiplier(self):
+        """Get Br multiplier.
+
+        Returns
+        -------
+        float
+        """
+        return self._br_multiplier
+
+    @br_multiplier.setter
+    def br_multiplier(self, br_multiplier):
+        self._br_multiplier = br_multiplier
+
+    @property
+    def br_value(self):
+        """Br value of magnet before mag factor applied.
```

Comment:
Don't use abbreviations in descriptions. Spell out "mag" rather than abbreviate it.